store/copr/coprocessor.go

```diff
@@ -752,6 +757,19 @@ func (worker *copIteratorWorker) logTimeCopTask(costTime time.Duration, task *co
 	if resp.Resp != nil {
 		switch r := resp.Resp.(type) {
 		case *coprocessor.Response:
+			selResp := &tipb.SelectResponse{}
+			proto.Unmarshal(r.Data, selResp)
+			if selResp != nil && len(selResp.GetExecutionSummaries()) > 0 {
+				readExecution := selResp.GetExecutionSummaries()[0]
+				affectRow := int(readExecution.GetNumProducedRows())
+				readTime := float64(readExecution.GetTimeProcessedNs()) / float64(time.Millisecond)
+				readByte := r.GetExecDetailsV2().GetScanDetailV2().GetReadBytes()
+				if affectRow < 20 && readByte < 1*1024*1024 {
+					metrics.TiKVSmallReadDuration.Observe(readTime)
+				} else {
+					metrics.TiKVLargeReadThroughput.Observe(float64(readByte) / readTime)
+				}
```

Comment:
How about extract this to a function? since it is duplicate with `store/tikv/tikvrpc/tikvrpc.go#938`